api.md

```diff
@@ -170,6 +172,118 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## add_data_node() :community_function: [](add_data_node)
+
+Add a new data node to the database to be used for creating
+distributed hypertables.
+
+Newly created distributed hypertables will be able to make use of this
+data node for distributing data across the cluster. Note that existing
+distributed hypertables will not use the newly added data node. For
+existing distributed hypertables to use added data nodes, you need to
+use [`attach_data_node()`](#attach_data_node).
+
+The function will attempt to bootstrap the data node by creating the
+database where the remote hypertables will later be created and also
+load the TimescaleDB extension on the data node.
+
+#### Errors
+
+* An error will be given if an attempt is made to run the function
+  inside a transaction.
+* An error will be given if the remote database already exists on the
+  data node and `if_not_exists` is not set.
+
+#### Required Arguments [](add_data_node-required-arguments)
+
+| Name        | Description            |
+| ----------- | -----------            |
+| `node_name` | Name of the data node. |
+
+##### Parameter Description
+
+* `node_name` - Name to use for the node locally.
+
+#### Optional Arguments [](add_data_node-optional-arguments)
+
+| Name                 | Description                                           |
+|----------------------|-------------------------------------------------------|
+| `host`               | Host name for the remote data node.                   |
+| `port`               | Port to use on the remote data node.                  |
+| `database`           | Database name where the remote hypertable is created. |
+| `local_user`         | User name on the local node.                          |
+| `remote_user`        | User name on the remote data node.                    |
+| `password`           | Password on the remote data node.                     |
+| `if_not_exists`      | Do not fail if the data node already exists.          |
+| `bootstrap_database` | Database to bootstrap.                                |
+| `bootstrap_user`     | User to be used for bootstrapping.                    |
+| `bootstrap_password` | Password to be used when bootstrapping.               |
+
+##### Parameter Description
+
+* `host` - the name of the host to use for the remote data node. This
+  defaults to `'localhost'`.
+
+* `port` - Port to use for the remote data node. This defaults to the
```

Comment:
I think it actually defaults to the port used by the access node, which is usually the default postgres port

Reply:
Not sure what you want changed here.

Reply:
I'm thinking he's suggesting that rather than defauling to default PostgreSQL port, it's:  ```* `port` - Port to use for the remote data node. This defaults to the
  port used by the access node```
or 
```by the access node on which the function is executed```

Reply:
Went with "The default is the  PostgreSQL port used by the access node on which the function is  executed."